plugins/doors/entities/weapons/nut_keys.lua

```diff
@@ -1,164 +1,126 @@
+    if not IsValid(self.Owner) then return end
+    local viewModel = self.Owner:GetViewModel()
+    if IsValid(viewModel) then end --viewModel:SetPlaybackRate(1) --viewModel:ResetSequence(ACT_VM_FISTS_DRAW)
 
-	if (IsValid(viewModel)) then
-		--viewModel:SetPlaybackRate(1)
-		--viewModel:ResetSequence(ACT_VM_FISTS_DRAW)
-	end
-
-	return true
+    return true
 end
 
 function SWEP:Holster()
-	if (!IsValid(self.Owner)) then
-		return
-	end
-
-	local viewModel = self.Owner:GetViewModel()
+    if not IsValid(self.Owner) then return end
+    local viewModel = self.Owner:GetViewModel()
 
-	if (IsValid(viewModel)) then
-		viewModel:SetPlaybackRate(1)
-		viewModel:ResetSequence(ACT_VM_FISTS_HOLSTER)
-	end
+    if IsValid(viewModel) then
+        viewModel:SetPlaybackRate(1)
+        viewModel:ResetSequence(ACT_VM_FISTS_HOLSTER)
+    end
 
-	return true
+    return true
 end
 
 function SWEP:Precache()
 end
 
 function SWEP:Initialize()
-	self:SetHoldType(self.HoldType)
+    self:SetHoldType(self.HoldType)
 end
 
 function SWEP:PrimaryAttack()
-	local time = nut.config.get("doorLockTime", 1)
-	local time2 = math.max(time, 1)
-
-	self:SetNextPrimaryFire(CurTime() + time2)
-	self:SetNextSecondaryFire(CurTime() + time2)
-
-	if (!IsFirstTimePredicted()) then
-		return
-	end
-
-	if (CLIENT) then
-		return
-	end
-
-	local data = {}
-		data.start = self.Owner:GetShootPos()
-		data.endpos = data.start + self.Owner:GetAimVector()*96
-		data.filter = self.Owner
-	local entity = util.TraceLine(data).Entity
-
-	if (hook.Run("KeyLockOverride",self:GetOwner(),entity)) then
-		return
-	end
-
-	--[[
+    local time = nut.config.get("doorLockTime", 1)
+    local time2 = math.max(time, 1)
+    self:SetNextPrimaryFire(CurTime() + time2)
+    self:SetNextSecondaryFire(CurTime() + time2)
+    if not IsFirstTimePredicted() then return end
+    if CLIENT then return end
+    local data = {}
+    data.start = self.Owner:GetShootPos()
+    data.endpos = data.start + self.Owner:GetAimVector() * 96
+    data.filter = self.Owner
+    local entity = util.TraceLine(data).Entity
+
+    --[[
 		Locks the entity if the contiditon fits:
 			1. The entity is door and client has access to the door.
 			2. The entity is vehicle and the "owner" variable is same as client's character ID.
 	--]]
-	if (IsValid(entity) and
-		(
-			(entity:isDoor() and entity:checkDoorAccess(self.Owner)) or
-			(entity:IsVehicle() and entity.CPPIGetOwner and entity:CPPIGetOwner() == self.Owner)
-		)
-	) then
-		self.Owner:setAction("@locking", time, function()
-			self:toggleLock(entity, true)
-		end)			
+    if IsValid(entity) and ((entity:isDoor() and (entity:checkDoorAccess(self.Owner) or (self.Owner:IsAdmin() ))) or (entity:IsVehicle() and ((entity.CPPIGetOwner and entity:CPPIGetOwner() == self.Owner) or (entity.GetCreator and entity:GetCreator() == self.Owner)))) then
+        self.Owner:setAction("@locking", time, function()
+            self:toggleLock(entity, true)
+        end)
```

Comment:
Remove redundant parentheses to simplify the conditional statements.
Indent the nested if statements for better readability.
Reformat the code to adhere to common Lua coding conventions.
Remove unnecessary spaces and align the code for consistency.